server/sequelize-data.json

```diff
@@ -1,4 +1,6 @@
 [
```

Comment:
Are these changes intentional? Maybe this should be removed from the PR?

Reply:
This is unintentional .